app/src/main/res/drawable/color_cursor.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
   <size android:width="0.5dp" />
-  <solid android:color="@color/oppia_primary_text" />
+  <solid android:color="@color/label_text_color" />
```

Comment:
I think instead of using `label_text_color` define another component color named `cursor_color` and use that instead as it would be add more clear context.

Reply:
Yeah, that will be better. Done.